internal/storage/fs/snapshot.go

```diff
@@ -626,19 +704,65 @@ func (ss *storeSnapshot) GetEvaluationDistributions(ctx context.Context, ruleID
 }
 
 func (ss *storeSnapshot) GetEvaluationRollouts(ctx context.Context, namespaceKey, flagKey string) ([]*storage.EvaluationRollout, error) {
-	panic("not implemented")
+	ns, ok := ss.ns[namespaceKey]
+	if !ok {
+		return nil, errs.ErrNotFoundf("namespaced %q", namespaceKey)
+	}
+
+	rollouts, ok := ns.evalRollouts[flagKey]
+	if !ok {
+		return nil, errs.ErrNotFoundf(`flag "%s/%s"`, namespaceKey, flagKey)
+	}
+
+	return rollouts, nil
 }
 
 func (ss *storeSnapshot) GetRollout(ctx context.Context, namespaceKey, id string) (*flipt.Rollout, error) {
-	panic("not implemented")
+	ns, err := ss.getNamespace(namespaceKey)
+	if err != nil {
+		return nil, err
+	}
+
+	rollout, ok := ns.rollouts[id]
+	if !ok {
+		return nil, errs.ErrNotFoundf(`rollout "%s/%s"`, namespaceKey, id)
+	}
+
+	return rollout, nil
```

Comment:
Ah yes, this reminds me we need to make an issue that this method is not really useable.
We can't create real random IDs that map across instances of Flipt.
The same applies to `GetRule`, it is a problem there too.

We either have (a) to come up with a consistent way of addressing these across Flipt instances, or (b) return not implemented.
(b) is tempting because the UI doesn't use these methods currently.

Reply:
@GeorgeMac Exactly. I noticed that during the writing of this impl. I couldn't really see how it is usable.